index.html

```diff
@@ -259,23 +275,7 @@ <h2>A project for first-time contributions</h2>
 
         <!-- COPY everything BELOW this, from contributor card start to end along with the "START" and "END" comment lines -->
 
-        <!-- ________ Contributor's card START ________  -->
```

Comment:
The problem is here. This is the card Template and you are only supposed to copy it. Instead you have completely removed it. You will need to return it to it's original state before I can accept your pull request. 
This should be very easy. For example you can take a look at the [Index.html file](https://github.com/Syknapse/Contribute-To-This-Project/blob/master/index.html) of this project, scroll down to the template section, copy it and add it back to your index.html on your computer. Save the changes and repeat steps 6 & 7 of the tutorial; commit and push your changes. And that should be it. I will see the changes and if everything is in order I will approve your push request. 
If you are not sure how to do it just ask me for help, it's a simple thing to solve and I'm sure you can do it 😉 
Good luck, and I'm waiting for the new commit.